parser.y

```diff
@@ -2620,7 +2620,7 @@ BeginTransactionStmt:
 |	"START" "TRANSACTION" "WITH" "CAUSAL" "CONSISTENCY"
```

Comment:
maybe make the statement <code>START TRANSACTION WITH CAUSAL CONSISTENCY **ONLY**</code> to emphasize it is a downgrade from linearizability.